README.md

```diff
@@ -0,0 +1,223 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+  <br/>
+  <br/>
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./public/logo3.svg" alt="logo" width="400"  height="auto" />
+  <br/>
+  <br/>
+  <br/>
+  <br/>
+
+  <h1><b>Metrics webapp</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Metrics webapp ](#-metrics-webapp-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Metrics webapp <a name="about-project"></a>
+
+Metrics webapp is a project with great potential.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+
+<details>
+  <summary>Client</summary>
+   - React <br>
+   - Redux Toolkit <br>
+   - Tailwind CSS <br>
+   - React Router <br>
+</details>
+
+<details>
+  <summary>Server</summary>
+    - Null
+</details>
+
+<details>
+<summary>Database</summary>
+    - Null
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Responsive Design**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- [Live Demo](https://649a4e85b14d052e99046154--tubular-toffee-d5af3c.netlify.app/)
+- [Proyect Overview Video](https://www.loom.com/share/06f9e71a283f4338b337bb958af273b8?sid=86295f38-2507-4ec4-b5ee-f6fa6f5e776c)
```

Comment:
- [ ] You have done a great job💪 on your video presentation; however🙆‍♀️, you're missing something very important: showing a piece of code👩‍💻 that you're proud of. 🙏Please go ahead and display a piece of code that you're proud of. Remember, in this case, you should show the actual code either from your text editor, GitHub, or any other suitable platform. The key point is to present the actual code, as this will help viewers👨‍👩‍👦‍👦 understand what you have accomplished and showcase your expertise.

Reply:
Thank you very much for the advice, I will try to make a better video showing part of the code, thanks for your help.